go/app/main.go

```diff
@@ -20,6 +21,22 @@ type Response struct {
 	Message string `json:"message"`
 }
 
+type Items struct {
+	Items []Item `json:"items"`
+}
+
+type Item struct {
+	Name     string `json:"name"`
+	Category string `json:"category"`
+}
+
+func sendError(c echo.Context, err_message string) error {
+	c.Logger().Debugf(err_message)
```

Comment:
```suggestion
	c.Logger().Errorf(err_message)
```
的なかんじでErrorはErrorログとして出してあげるとさらに良さそう！

Reply:
コメントありがとうございます！
`Errorf`というものがあるのですね！ログ出力に関して、使用できる関数・使い分け調べてみます;)